.github/workflows/xtest.yml

```diff
@@ -19,10 +19,6 @@ on:
         required: false
         type: string
         default: main
-      tests-ref:
-        required: false
-        type: string
-        default: main
```

Comment:
this breaks the pr workflow called in opentdf/tests -- it always ends up calling main instead of the pr ref.
it is not needed as one can just call a specific tests ref with `uses: opentdf/tests/.github/workflows/xtest.yml@my-ref`